### PR TITLE
Pieskiuw 23 moving to postgre sql

### PR DESCRIPTION
**Pull request by kamart02**
I've modified django settings to move to a remote postgresql database. Also i've added a script to create secrets file.

### CODE REVIEW

**Comment by MateuszWasilewski** [comment]
Looks good but I think you should squash some commits ex. one wirh commit msg "mend"

File: django/pieskiUW/settings.py [hunk 1]
```diff
@@ -12,6 +12,12 @@
 
 from pathlib import Path
 
+# pylint: disable=no-name-in-module
+
```
Comment (MateuszWasilewski) [inline django/pieskiUW/settings.py:16]: empty lines seem excesive

**Review by MateuszWasilewski — APPROVED** [review]
Looks solid. Minor nitpicking. Awesome contribution

**Review by jpalikowska — APPROVED** [review]

File: django/pieskiUW/settings.py (hunk 1 above)
Comment (kamart02) [inline django/pieskiUW/settings.py:16]:
```suggestion
```

File: django/pieskiUW/settings.py [hunk 2]
```diff
@@ -12,6 +12,12 @@
...
+from .secrets import postgresql
+
```
Comment (kamart02) [inline django/pieskiUW/settings.py:18]:
```suggestion
```

**Pull request merged**